packages/@lwc/perf-benchmarks/src/__benchmarks__/engine-dom/benchmark-expression/expression.benchmark.js

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright (c) 2018, salesforce.com, inc.
+ * All rights reserved.
+ * SPDX-License-Identifier: MIT
+ * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/MIT
+ */
+
+import { createElement } from '@lwc/engine-dom';
+
+import Expression from '@lwc/perf-benchmarks-components/dist/dom/benchmark/expression/expression.js';
+import Store from '@lwc/perf-benchmarks-components/dist/dom/benchmark/store/store.js';
+import { insertComponent, destroyComponent } from '../../../utils/utils.js';
+
+benchmark(`dom/expressions`, () => {
+    let expressionElement;
+
+    before(() => {
+        expressionElement = createElement('benchmark-expression', { is: Expression });
+        return insertComponent(expressionElement);
+    });
+
+    run(() => {
+        const store = new Store();
+        store.runLots();
```

Comment:
This is true of existing tests, but I think it was a mistake to put `store.runLots()` in `run()`. We don't need to measure the cost of allocating a gigantic array.

I've opened a separate issue to track this: https://github.com/salesforce/lwc/issues/3607